azurerm/internal/services/netapp/netapp_volume_resource.go

```diff
@@ -194,6 +194,43 @@ func resourceNetAppVolume() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+
+			"data_protection_replication": {
```

Comment:
is there a reason this is called `data_protection_replication` when the sdk property is `data_protection` (thou i'd argue data_replication or even replication might suffice)

Reply:
There will be other types of data_protection coming up, e.g. snapshotpolicy, backup, etc.. So keeping this naming will make it consistent with that customers see in the portal and other tools and will make sense to them.